apps/finance/app/src/components/Balances.js

```diff
@@ -1,152 +1,170 @@
-            overflow-x: auto;
-            ${noBalances
-              ? `
+          min-height: 112px;
+          transform: translate3d(0, 0, 0);
+          overflow-x: auto;
+          ${balanceItems.length === 0
+            ? `
                 display: flex;
                 justify-content: center;
                 align-items: center;
               `
-              : ''}
-          `}
-        >
-          {noBalances ? (
-            <div
-              css={`
-                ${textStyle('body1')};
-                color: ${theme.content};
-              `}
-            >
-              No token balances yet.
-            </div>
-          ) : (
-            <ul
-              css={`
-                list-style: none;
-                display: flex;
-                ${compactMode
-                  ? `
+            : ''}
+        `}
+      >
+        {balanceItems.length === 0 ? (
+          <div
+            css={`
+              ${textStyle('body1')};
+              color: ${theme.content};
+            `}
+          >
+            No token balances yet.
+          </div>
+        ) : (
+          <ul
+            css={`
+              list-style: none;
+              display: flex;
+              ${compactMode
+                ? `
                     flex-direction: column;
                     padding: ${1 * GU}px 0;
                   `
-                  : ''}
-              `}
-            >
-              {balanceItems.map(
-                ({ address, amount, convertedAmount, symbol, verified }) => (
-                  <li
-                    key={address}
-                    css={`
-                      display: block;
-                      min-width: ${20 * GU}px;
-                      ${compactMode ? `margin-bottom: ${3 * GU}px;` : ''}
-                      &:last-of-type {
-                        min-width: unset;
-                        margin-bottom: 0;
-                      }
-                    `}
-                  >
-                    <BalanceToken
-                      address={address}
-                      amount={amount}
-                      compact={compactMode}
-                      convertedAmount={convertedAmount}
-                      symbol={symbol}
-                      verified={verified}
-                    />
-                  </li>
-                )
-              )}
-            </ul>
-          )}
-        </div>
-      </Box>
-    )
-  }
+                : ''}
+            `}
+          >
+            {balanceItems.map(
+              ({
+                address,
+                amount,
+                convertedAmount,
+                decimals,
+                symbol,
+                verified,
+              }) => (
+                <li
+                  key={address}
+                  css={`
+                    flex-shrink: 0;
+                    display: block;
+                    min-width: ${20 * GU}px;
+                    padding-right: ${3 * GU}px;
```

Comment:
```suggestion
                    padding-right: ${4 * GU}px;
```

I was looking at the A1 org, and for larger token balances, the additional 8px is nice (since we have the min-width applied as well, this doesn't really affect smaller token balances).

Reply:
Agreed, especially with the new `<Help />` :+1: